### PR TITLE
DOC: add meegkit to related software suite

### PR DESCRIPTION
**Pull request by sappelhoff**

### CODE REVIEW

**Comment by larsoner** [comment]
Even better would be to add it to conda-forge, then to mne-installers. Then it will be automatically added. I'll see if I can get the ball rolling

File: doc/sphinxext/related_software.py [hunk 1]
```diff
@@ -97,11 +99,6 @@
         "Home-page": "https://emd.readthedocs.io/en/stable",
         "Summary": "Empirical Mode Decomposition in Python.",
     },
-    # Needs https://github.com/vferat/pycrostates/pull/188 and a release
```
Comment (larsoner) [inline doc/sphinxext/related_software.py:100]: ... but either way we should keep this PR open so we can benefit from this simplification

File: doc/sphinxext/related_software.py [hunk 2]
```diff
@@ -30,8 +30,10 @@
 # If it's available on PyPI, add it to this set:
 PYPI_PACKAGES = {
     "alphaCSC",
+    "meegkit",
     "meggie",
     "niseq",
+    "pycrostates",
```
Comment (larsoner) [inline doc/sphinxext/related_software.py:36]: It looks like we actually don't need it here, either

https://app.circleci.com/pipelines/github/mne-tools/mne-python/25808/workflows/eb3c3881-0351-4947-8aeb-ce7ee7a8a11f/jobs/69774

```suggestion
```

**Comment by larsoner** [comment]
Work in progress in https://github.com/conda-forge/staged-recipes/pull/28279 (and https://github.com/nbara/python-meegkit/pull/81)

**Comment by sappelhoff** [comment]
Great, thanks @larsoner

**Comment by sappelhoff** [comment]
Great, so one off the manual list, and another one on :)

**Pull request merged**